### PR TITLE
assignment tasks completed

### PR DESCRIPTION
**Pull request by anupgp**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Added code to complete the assignment
## What did you learn from the changes you have made?
I learned to perform one of the simplest of supervised classification models using KNN
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I tried to do everything by coding from scratch. But, using scikit-learn module simplifies and makes the whole-process more efficient and easy to understand
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
Setting up cross-validation and fine-tuning the model parameters was bit challenging but not difficult.
## How were these changes tested?
I measured accuracy of the model as well as looked at the classification visually
## A reference to a related issue in your repository (if applicable)
NA
## Checklist
- [ x] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by fyellow — COMMENTED** [review]
Reviewed!
